src/tox/config/loader/str_convert.py

```diff
@@ -31,8 +31,7 @@ def to_list(value: str, of_type: Type[Any]) -> Iterator[str]:
 
     @staticmethod
     def to_set(value: str, of_type: Type[Any]) -> Iterator[str]:
-        for value in StrConvert.to_list(value, of_type):
-            yield value
+        yield from StrConvert.to_list(value, of_type)
```

Comment:
nitpick: While this technically is a shorter version of it, I find the left side is easier to debug because you actually can put a breakpoint there, while in this case, you're out of luck